File: COVID19/main.py
from bs4 import BeautifulSoup
from urllib.request import urlopen
import requests
import time

site = "http://ncov.mohw.go.kr/bdBoardList_Real.do?brdId=1&brdGubun=12&ncvContSeq=&contSeq=&board_id=&gubun="

page = requests.get(site)
bsObject = BeautifulSoup(page.text, "html.parser")
parseLoc = bsObject.find_all('td')

parseLoc = str(parseLoc).replace("<td>","").replace("</td>","\n").replace("<br/>","\n")
lists = []
lists = parseLoc.split('\n')
for tst in lists:
    time.sleep(1)
    print(tst)

File: COVID19/nodeJS/main.js
const axios = require("axios");
const cheerio = require("cheerio");
const iconv = require("iconv-lite");

var url = {
  url:
    "http://ncov.mohw.go.kr/bdBoardList_Real.do?brdId=1&brdGubun=12&ncvContSeq=&contSeq=&board_id=&gubun=",
  method: "GET",
  responseType: "arraybuffer",
  responseEncoding: "binary",
};

const getHtml = async () => {
  try {
    const response = await axios(url);
    const html = iconv.decode(response.data, "utf-8").toString();
    return html;
  } catch (error) {
    console.error(error);
  }
};

getHtml().then((html) => {
  var tdList = [];
  var COVIDLoc = [];
  var now = 0;
  const $ = cheerio.load(html);
  const $tbodyList = $("div.box_line2 table.midsize.big tbody tr td");

  $tbodyList.each(function (i, element) {
    tdList[i] = $(this)
      .text()
      .replace(/\(/g, "\n")
      .replace(/\)/g, "")
      .split("\n");
  });

  for (var i = 2; i < tdList.length; i += 5) {
    COVIDLoc[now] = {
      name: tdList[i][0],
      location: tdList[i][1],
      date: tdList[i + 1][0],
    };

    now++;
  }
  console.log(COVIDLoc);
});
